apps.yaml

```diff
@@ -97,3 +97,15 @@ HELAO:
   categories:
     - technology-lab
     - technology-active-learning
+prisma:
+  metadata:
+    title: PRISMA
+    description: |
+        An app for for high-throughput analysis of spectra. More details in the app homepage and peer-reviewed article:
```

Comment:
Do you want to add a link to the article or is the link provided under `external_url` sufficient?

Reply:
The external link is sufficient. The scientific paper is accepted but it does not have DOI yet. I will add the DOI directly in the site under external_url

Reply:
Thanks! Now corrected, also the 'An app for for' i saw there.